internal/service/streamprocessor/data_source_test.go

```diff
@@ -10,22 +10,32 @@ import (
 )
 
 var dataSourceName = "data.mongodbatlas_streamprocessor.test"
+var pluralDataSourceName = "data.mongodbatlas_streamprocessors.test"
```

Comment:
q: would we eventually move these data source test with basic test of resource?

Reply:
Yes. But got spot of the naming. Changing it now

Reply:
Adjusted in 8345bb61353b1580ffbd3b0cb4f412fa4acdb1a5